### PR TITLE
move it ! :)

### PR DESCRIPTION
**Pull request by blackcrack**
Hi,

i have move the menu more in right direction
and have remove the space between the front
to be better skinable and to be not over the title of the heading :)
now fit's also for skinning :)
best regards
Blacky

### CODE REVIEW

**Comment by blackcrack** [comment]
and have update the commit again, because,
to be more fail free ;)

best regards
Blacky

File: templates/direct_menu.php [hunk 1]
```diff
@@ -26,9 +26,13 @@
 
 	#navigation div ul {
 		position: relative;
-		margin-left: 170px;
+		padding-right:10px;
```
Comment (juliusknorr) [inline templates/direct_menu.php:29]: What is the reason for the right padding? Should work fine without. :wink:

File: templates/direct_menu.php [hunk 2]
```diff
@@ -26,9 +26,13 @@
...
 	}
 
+       div#navigation.menu {
+                margin-left: 170px;
```
Comment (juliusknorr) [inline templates/direct_menu.php:33]: :+1: Makes sense to have the margin with the #navigation element

**Comment by juliusknorr** [comment]
@blackcrack Thanks for the contribution, although i don't get the original problem with the direct_menu app, moving the margin to the #navigation element seems fine to me. Can you comment on why the  padding-right is required?

**Review by juliusknorr — CHANGES_REQUESTED** [review]

File: templates/direct_menu.php (hunk 1 above)
Comment (blackcrack) [reply templates/direct_menu.php:29]: was an less space between the icons and have not look very nice, so with 10 px by side, look's better, uniform and smooth.. try it out, be just 10 px, but this maks's it.. :)

best regards
blacky

File: templates/direct_menu.php (hunk 2 above)
Comment (blackcrack) [reply templates/direct_menu.php:33]: thy :)

File: templates/direct_menu.php (hunk 1 above)
Comment (juliusknorr) [reply templates/direct_menu.php:29]: the padding right on the `<ul>` element is not affecting the space between the icons, it is just the space after the last icon, which has no influence on the vanilla nextcloud theme. :wink: See comments on #54 on how you can integrate this withing your theme.

**Comment by juliusknorr** [comment]
Closing since direct_menu will not be developed further.

**Pull request closed without merging**